examples/hcpb_radial_build_example/HCPB_Build_Dict.py

```diff
@@ -0,0 +1,135 @@
+    "fw": {
+        "thickness": 3.8,
+        "colour": "#DAA520",  # Goldenrod
+        "scores": ["fast_fluence", "he_prod", "fe_dpa", "heating"],
+    },
+    "breeder": {
+        "thickness": 50,
+        "description": "Thickness Varies",
+        "colour": "#B22222",  # Firebrick
+        "scores": ["tbr", "heating"],
+    },
+    "bw": {
+        "thickness": 10,
+        "colour": "#8B0000",  # Dark red
+        "scores": ["tbr", "heating","fast_fluence"],
+    },
+
+    "manifold": {
+        "thickness": 20,
+        "colour": "#5F9EA0",  # Cadet blue
+    },
+    
+    "hts": {
+        "thickness": 10.5,
+        "description": "Thickness Varies",
+        "colour": "#00FF00",  # Green
+        "scores": ["heating"],
+    },
+    
+    "gap_1": {
+        "thickness": 1,
+        "colour": "#F5F5F5",  # White smoke
+    },
+    
+    "vv_fill": {
+        "thickness": 10,
+        "colour": "#778899",  # Light slate grey
+        "scores": ["heating"],
+    },
+    
+    "gap_2": {
+        "thickness": 2,
+        "colour": "#E0FFFF",  # Light cyan
+    },
+    
+    "lts": {
+        "thickness": 10.5,
+        "description": "Thickness Varies",
+        "colour": "#00FA9A",  # Medium spring green
+        "scores": ["heating"],
+    },
+    
+    "thermal_insulator": {
+        "thickness": 10,
+        "colour": "#B0E0E6",  # Powder blue
+        "scores": ["heating"],
+    },
+    
+    "coil_pack": {
+        "thickness": 53,
+        "colour": "#FF4500",  # Orange red
+        "scores": ["heating"],
+    },
+}
+    for component in list(build_dict.keys()):
+        if component in list(mat_info.keys()):
+            build_dict[component]['composition'] = mat_info[component]['composition']
+            build_dict[component]['material_name'] = component
+    return build_dict
+    
+def make_toroidal_model(materials_xml_file, major_radius, minor_radius_z, minor_radius_xy, build_dict):
+        # openmc Materials object:
+        materials = openmc.Materials.from_xml(materials_xml_file)
+        toroidal_model = ToroidalModel(
+        build_dict, major_radius, minor_radius_z, minor_radius_xy, materials)
+        mat_geom, cell_dict = toroidal_model.get_openmc_model()
+        return mat_geom, cell_dict
+
+def plot_radial_build(radial_plot_name, build_dict):
+        radial_build_plot = RadialBuildPlot(build_dict, title=radial_plot_name, size=(15, 5))
+        radial_build_plot.plot_radial_build()
+        radial_build_plot.to_png()
+        return radial_build_plot
+
+def main():
+    def parse_args():
+        parser = argparse.ArgumentParser()
+        parser.add_argument('--HCPB_YAML', default = 'HCPB_YAML.yaml', help="Path (str) to YAML containing inputs for HCPB build dictionary & mix materials")
+        args = parser.parse_args()
+        return args
+    
+    def read_yaml(args):
+        with open(args.HCPB_YAML, 'r') as hcpb_yaml:
+            yaml_inputs = yaml.safe_load(hcpb_yaml)
+        return yaml_inputs
```

Comment:
These functions can be defined outside of `main` at the global scope.  I'm not sure I have a clear reason for this other than reduced complexity.  I think it is rare to have functions defined within other functions, and usually serves a very specific purpose.